ocsci/pytest_customization/ocscilib.py

```diff
@@ -126,6 +164,10 @@ def process_cluster_cli_params(config):
     ocsci_config.RUN['cli_params']['deploy'] = get_cli_param(config, "deploy", default=False)
     ocsci_config.ENV_DATA['cluster_name'] = cluster_name
     ocsci_config.ENV_DATA['cluster_path'] = cluster_path
+    ocsci_config.ENV_DATA['email'] = get_cli_param(config, 'email')
+    ocsci_config.ENV_DATA['report'] = get_cli_param(config, '--html')
```

Comment:
Why here is used the name of the cli parameter with dash: `--html`? (while on previous line it is `email` without the dash prefix)

Reply:
html options is coming from pytest-html unlike our own customize command line option. 

```
html = config.getoption('html')
    print("html: %s" % html)
```
error: 
INTERNALERROR>     raise ValueError("no option named %r" % (name,))
INTERNALERROR> ValueError: no option named 'html'

If we use --html, its working as expected